extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/local_inference/automatic_speech_recognition.py

```diff
@@ -198,7 +196,6 @@ def refine_pipeline_creation_params(model_settings: Dict[str, Any]) -> List[Dict
     """
 
     supported_keys = {
-        "model",
```

Comment:
Not blocking for this diff but removing this would now mean deserialize won't return "model" back, so in the case where it's called by itself without calling "run", no model setting would be present.

I think the alternative is to also call get_hf_model in deserialize and pass it in to this function.

But that can be done at top of stack

Reply:
I don't think any of the other HF model parsers have "model" as a supported key, because that's technically not one of the completion params passed into the HF `__call__` methods. Added a note of this in https://github.com/lastmile-ai/aiconfig/issues/822

Reply:
Yep, understood. However this specific HF __call__ method does accept `model`, so `refine_completion_params` for this model parser should have it as a supported key, and set it properly based on the model name/settings

Reply:
Got it, made a note of that in the issue above